test-cases/javascript/interactive/listener-and-onclick.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
```

Comment:
It doesn't necessarily have to be `onclick` so maybe the file could be named `event-listener-and-event-attribute`?